src/main/java/frc/robot/commands/autonomous/SmashAndDash.java

```diff
@@ -0,0 +1,221 @@
+package frc.robot.commands.autonomous;
+
+import com.pathplanner.lib.PathConstraints;
+import com.pathplanner.lib.PathPlanner;
+import com.pathplanner.lib.PathPlannerTrajectory;
+import com.spikes2212.command.drivetrains.commands.DriveArcade;
+import com.spikes2212.command.genericsubsystem.commands.smartmotorcontrollergenericsubsystem.MoveSmartMotorControllerGenericSubsystem;
+import com.spikes2212.dashboard.RootNamespace;
+import com.spikes2212.util.UnifiedControlMode;
+import edu.wpi.first.wpilibj2.command.*;
+import frc.robot.commands.*;
+import frc.robot.commands.PlaceGamePiece.ArmState;
+import frc.robot.services.ArmGravityCompensation;
+import frc.robot.services.VisionService;
+import frc.robot.subsystems.*;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Supplier;
+
+public class SmashAndDash extends BasePathAuto {
```

Comment:
![Untitled](https://github.com/Spikes-2212-Programming-Guild/ChargedUp2023/assets/71877371/621753e6-faf4-4e30-965c-43b0915bf683)

by the way, the namespace things were for testing, if you care enough - delete it.
also there are a lot of magic numbers here (despite the existence of more than 15 constants)